Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -364,6 +364,15 @@
       head:
       - state: lizard-equipped-HELMET
         shader: unshaded
+  - type: Item
+    heldPrefix: rainbowlizardplush
```

Comment:
This isn't correct, you need to set this on the `Sprite` component instead.

Reply:
rainbowcarpplush also has this problem, let me know if you want it fixed as well

requested changes pushed